src/App.js

```diff
@@ -1,7 +1,25 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import { Navbar, Sidebar } from "./components";
+import { Home, Label } from "./pages";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <div className="page-wrapper">
+        <Navbar />
+
+        <section className="main-section">
+          <Sidebar />
+
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/label" element={<Label />} />
+          </Routes>
```

Comment:
You can make a separate component for routes.

Reply:
Done, thank you!